application/gqlgen/schema.graphql

```diff
@@ -376,3 +514,79 @@ input RemoveOrganizationTrustInput {
     primaryID: ID!
     secondaryID: ID!
 }
+
+"""
+Invitation to a `Meeting`. An invitation must be confirmed by the invitee before they may be added to a `Meeting`.
+"""
+type MeetingInvitation {
+    "ID of the `Meeting`"
+    meetingID: ID!
+    "ID of the `User` making the invitation"
+    userID: ID!
+    "Email address of the invitee"
+    email: String!
+    "Gravatar image URL. Always a valid URL, but depending on the email address, it may reference a generic avatar."
+    avatarURL: String!
```

Comment:
What is this for - the invitee?  Should it at least be optional?

Reply:
Yes, it's for the invitee. No, because the UI needs to display something, even if it's just a generic graphic.